frontend/components/common/unauthorised-access.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from "../ui/button";
```

Comment:
One more change before merging, we should try to use absolute path for all  imports instead of relying on relative paths, because components like this can be moved around. So I propose to use `@/components/ui/button` instead as defined in the `components.json` file in the root directory.